src/mlir/verona-mlir.cc

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "CLI/CLI.hpp"
+#include "ast/cli.h"
+#include "ast/module.h"
+#include "ast/parser.h"
+#include "ast/path.h"
+#include "ast/prec.h"
+#include "ast/ref.h"
+#include "ast/sym.h"
+#include "dialect/VeronaDialect.h"
+#include "generator.h"
+#include "mlir/InitAllDialects.h"
+#include "mlir/InitAllPasses.h"
+
+#include "llvm/Support/InitLLVM.h"
+
+namespace
+{
+  // Source file types, to choose how to parse
+  enum class Source
+  {
+    NONE,
+    VERONA,
```

Comment:
Why are these capitalised?  Other enums are not.

Reply:
no reason, will change